doc/rados/operations/pools.rst

```diff
@@ -229,8 +245,8 @@ pool by running the following command.:
 Setting Pool Quotas
 ===================
 
-To set pool quotas for the maximum number of bytes and/or the maximum number of
-RADOS objects per pool, run the following command:
+To set quotas for the maximum number of bytes and/or the maximum number of
```

Comment:
Can we get away with "and" in place of "and/or"?

Reply:
They don't need to be set together, so an admin might set just one or the other.  I'm pushing a reword, what do you think of

>To set a quota for the maximum number of bytes or the maximum number of
RADOS objects per pool, run a command of the following form:

>To remove a quota, set its value to ``0``.  Note that you may set a quota only
for bytes or only for RADOS objects, or can set both.

Reply:
I like it.

Reply:
I'd meant "or you can"

Reply:
following up in a fresh PR